torchgen/gen.py

```diff
@@ -536,7 +536,7 @@ class RegisterSchema:
     def __call__(self, f: NativeFunction) -> Optional[str]:
         if not self.selector.is_native_function_selected(f):
             return None
-        tags = "{" + ", ".join([f"at::Tag::{tag}" for tag in f.tags]) + "}"
+        tags = "{" + ", ".join((f"at::Tag::{tag}" for tag in sorted(f.tags))) + "}"
```

Comment:
You can remove the parenthesis altogether here as this is a generator anyways right?